spynnaker/pyNN/models/neural_projections/connectors/from_list_connector.py

```diff
@@ -235,6 +236,14 @@ def __repr__(self):
     def conn_list(self):
         return self.__conn_list
 
+    @property
+    def split_conn_list(self):
```

Comment:
It might make more sense to make this something like "get_n_connections(pre_slices, post_slices, pre_slice, post_slice)".  Then you can create the split_conn_list in here if it doesn't already exist.